src/app/index.run.js

```diff
@@ -15,17 +15,21 @@
     });
 
     $rootScope.$on('$locationChangeSuccess', function(e) {
-      if (GData.isLogin()) {
+      var firebaseDriver = $injector.get('firebaseDriver');
```

Comment:
Ah, I don't know. I think I tried just injecting `firebaseDriver`, but something errored and I got scared. I'll try again!